dissect/target/plugins/os/unix/generic.py

```diff
@@ -20,6 +20,8 @@ def activity(self) -> Optional[datetime]:
 
         last_seen = 0
         for f in var_log.iterdir():
+            if f.is_symlink() or not f.exists():
```

Comment:
Why outright skip symlinks? Would you not just want to check for the file existence?

Reply:
Correct, the `is_symlink` check is unnecessary. Removed in  [ffb3e36](https://github.com/fox-it/dissect.target/pull/469/commits/ffb3e367f2cc10ff79c2b78c1c4c30df85c9c681).